app/scripts/modules/core/application/modal/groupMembershipConfigurer.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="form-group row" ng-if="$ctrl.fiatEnabled">
+  <div class="col-sm-3 sm-label-right">Group Membership</div>
+  <div class="col-sm-2">
+    <ui-select multiple
+               tagging
+               tagging-label="&zwnj;"
```

Comment:
Are these groups required to read or required to write?
On Tue, Sep 6, 2016 at 12:56 PM Chris Berry <notifications@github.com>
wrote:

> In
> app/scripts/modules/core/application/modal/groupMembershipConfigurer.component.html
> <https://github.com/spinnaker/deck/pull/2662#discussion_r77706588>:
>
> > @@ -0,0 +1,14 @@
> > +<div class="form-group row" ng-if="$ctrl.fiatEnabled">
> > +  <div class="col-sm-3 sm-label-right">Group Membership</div>
> > +  <div class="col-sm-2">
> > +    <ui-select multiple
> > +               tagging
> > +               tagging-label="&zwnj;"
>
> ?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/spinnaker/deck/pull/2662/files/d6bcdbcdb96ff4fd313a10419969d21c8f45ba15#r77706588>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAXuLIPLP2lpklvFjFGEvaEwV_GcWsAaks5qncV_gaJpZM4J2OBY>
> .
>